src/main/java/io/citrine/jcc/search/core/result/BaseMultiSearchResult.java

```diff
@@ -55,17 +55,30 @@ public Long getTook() {
     }
 
     /**
+     * Set the list of successes.  This overwrites any successes that are already saved.
+     * @param successes List of successes.
+     * @return This object.
+     */
+    @JsonSetter("successes")
```

Comment:
How about `BaseMultiSearchResult.getResult(int index)` and `BaseMultiSearchResult.getIsSuccess(int index)`, which could then be backed with a list of tuples?